src/hardhat/contracts/Staking/FraxUnifiedFarm_ERC20_V2.sol

```diff
@@ -763,8 +700,9 @@ contract FraxUnifiedFarm_ERC20_V2 is FraxUnifiedFarmTemplate_V2 {
         return(_safeTransferLockedByLockIndex([msg.sender, receiver_address], sender_lock_index, transfer_amount, use_receiver_lock_index, receiver_lock_index));
     }
 
+    /// @notice The transfer logic that executes the transfer, utilizes beforeLockTransfer & onLockReceived to ensure that the receiver is able to prevent asset loss
     function _safeTransferLockedByLockIndex(
```

Comment:
Added this to avoid the stack-to-deep issue

Reply:
shouldn't have a stack too deep issue here? wasn't running into this on my end, but if it is what is needed, so be it :) shouldn't impact things externally, since this is an internal change.

Reply:
I didn't want to miss any of the other little changes I had made, so I just synced my fork & cleaned it up to match the current file structure. Pull 183 "logic checks" should be the one to use:
https://github.com/FraxFinance/frax-solidity/pull/183/files#diff-d674aac7a787f8ee55c9411cb6de2ea4a8f350970454fabcebfd475bd69dd311